test/runnable/literal.d

```diff
@@ -225,13 +225,24 @@ void test13907()
     assert(aa13999[key13999] == "bar");
 }
 
+ulong op12950(ulong v){return v + 12950;}
+
+void test12950()
+{
+    assert(0x00_00_00_01.op12950() == 12951);
+    assert(0x00_00_00_01UL.op12950() == 12951);
+    assert(0b00_00_00_01.op12950() == 12951);
+    assert(0b00_00_00_01UL.op12950() == 12951);
+}
```

Comment:
Does this need to be a runnable test? They have more overhead.

Reply:
Yeah but it was the overhead of 1 more test case VS overhead of a few code in an existing one.

Reply:
Can't you use `static assert` and put it in `compilable`? Or even better do something like this: https://github.com/dlang/dmd/pull/9963.

Reply:
I think this is nice. The file already exists and from the semantic point of view what's added is very small.

Reply:
If the bug fix is in the lexer it doesn't make sense to run the rest of the compiler. Even less compile a executable and run it. Writing a test that only uses the lexer would make the bug fix clear that it only depend on the lexer.